host/sgx/linux/xstate.c

```diff
@@ -6,26 +6,17 @@
 #include <openenclave/internal/trace.h>
 #include "../cpuid.h"
 
-/*
-**==============================================================================
-**
-** Implementation of SGX Platform Capability Discovery.
-** Example usage is to set SECS.ATTRIBUTES correctly
-**
-**==============================================================================
-*/
+#define XSAVE_SHIFT 26
+#define OSXSAVE_SHIFT 27
+#define AVX_SHIFT 28
+bool is_xgetbv_supported(void);
```

Comment:
Same comment as for `_is_avx_supported`: prefix with `_` for local functions.

Also, the forward declaration here is unnecessary since it's defined immediately after this.

Reply:
Got a compiler error and so had to include the declaration. Converted this to static function with the  prefix _